dask/dataframe/io/parquet/arrow.py

```diff
@@ -72,7 +72,7 @@ def _write_partitioned(table, root_path, partition_cols, fs, index_cols=(), **kw
             subgroup, preserve_index=preserve_index, schema=subschema, safe=False
         )
         prefix = fs.sep.join([root_path, subdir])
-        fs.mkdir(prefix, exists_ok=True)
+        fs.mkdirs(prefix, exists_ok=True)
```

Comment:
Should the second argument be `exist_ok=True`?

Reply:
haha I didn't realize there were two different parts wrong :)

Reply:
I guess that means I should write a test...

Reply:
I was being silly. It is already failing tests :facepalm:

Reply:
Good! Didn't check the logs, but it's very nice that it covered it already.